Cargo.toml

```diff
@@ -2,6 +2,7 @@
 members = [
   "api",
   "cli",
+  "cli-tests",
```

Comment:
It's a fairly common pattern to split out a crate that's just for tests. For example, see [`tokio`](https://github.com/tokio-rs/tokio).

Reply:
That's interesting. How does this typically work with code owners? I assume that both directories will be marked as owned by the same team / group of individuals?

Reply:
Forgot to mention that splitting out test crates is necessary when enabling different features for tests.

I don't see people using code owners that often, so I'm not exactly sure. Assigning code owners can be as simple as
```codeowners
cli*/ @cli-team
```
or

```codeowners
cli/ @cli-team
cli-*/ @cli-team
```